RealmSwift/Realm.swift

```diff
@@ -719,21 +753,25 @@ public final class Realm {
     /**
      Performs actions contained within the given block inside a write transaction.
 
-     If the block throws an error, the transaction will be canceled, reverting any writes made in the block before
-     the error was thrown.
+     If the block throws an error, the transaction will be canceled and any
+     changes made before the error will be rolled back.
 
-     Write transactions cannot be nested, and trying to execute a write transaction on a Realm which is already
-     participating in a write transaction will throw an error. Calls to `write` from `Realm` instances in other threads
-     will block until the current write transaction completes.
+     Only one write transaction can be open at a time for each Realm file. Write
```

Comment:
yeah, ok. Hard to come up with phrasing that is generalized, accurate and not misleading. Fine to keep as-is.